sigar_interface_test.go

```diff
@@ -119,6 +119,37 @@ var _ = Describe("Sigar", func() {
 		}
 	})
 
+	It("net connection lists", func() {
+		connLists := []Getter{
```

Comment:
nit: maybe comment above this list "Implemented in Windows and Linux," and then above the other `connList` say "Implemented only in Linux"